webapp/src/utils/user_utils.js

```diff
@@ -18,3 +18,7 @@ export function displayUsernameForUser(user, config) {
 
     return '';
 }
+
+export function getErrorMessage(str) {
+    return JSON.parse(str);
```

Comment:
If we have it done this way, then the caller can just expect to receive a string.

```suggestion
	try {
		const parsed = JSON.parse(str);
		return parsed.message;
	} catch (e) {
    	return str;
    }
```